druid/examples/cursor.rs

```diff
@@ -61,11 +45,17 @@ impl<W: Widget<AppState>> Controller<AppState, W> for CursorArea {
         data: &mut AppState,
         env: &Env,
     ) {
-        if data.custom.is_none() {
-            data.custom = ctx.window().make_cursor(&data.custom_desc).map(Arc::new);
-        }
-        if matches!(event, Event::MouseMove(_)) {
-            ctx.set_cursor(&data.cursor);
+        match event {
+            Event::WindowConnected => {
+                let cursor_image =
+                    ImageBuf::from_data(include_bytes!("./assets/PicWithAlpha.png")).unwrap();
+                let custom_desc = CursorDesc::new(cursor_image, (0.0, 0.0));
+                data.custom = ctx.window().make_cursor(&custom_desc).map(Rc::new);
+            }
+            Event::MouseMove(_) => {
+                ctx.set_cursor(&data.cursor);
```

Comment:
I would move a bit of the docs from up top down here, and would say something like,

```
/// The cursor is reset by Druid at the start of each `MouseMove` event;
/// If we wish to use a non-default cursor we have to override this explicitly.

Reply:
Makes sense, I should have probably asked this in zulip but I was wondering if it was actually needed _every_ mouse move event. I will put something like that in.

Reply:
I think this is really valuable thing about having you look at the examples: if something is confusing, it will likely be confusing to other readers, and it means we should explain it. Please feel encouraged to ask any questions that occur to you!